exo-run/src/docker-runner.ls

```diff
@@ -63,11 +65,20 @@ class DockerRunner extends EventEmitter
         @emit 'online'
 
 
-  _check-dependency-containers: ~>
-    for dependency of @docker-config.dependencies
-      app-dependency = "#{@docker-config.app-name}-#{dependency}"
-      DockerHelper.ensure-container-is-running app-dependency, dependency
-      @docker-config.env[dependency.to-upper-case!] = DockerHelper.get-docker-ip app-dependency
+  _check-dependency-containers: (done) ~>
+    dependencies = []
+    for image, vars of @docker-config.dependencies
+      container-name = "#{@docker-config.app-name}-#{image}"
+      if vars.docker_flags?
+        volume = Handlebars.compile(that.volume)({"EXO_DATA_PATH": "/Users/originate/Desktop/data"})
```

Comment:
Correct, the one I used here was just a placeholder for testing it out locally. This prevents tests from passing on CI.